### DISCUSSION TITLE
When a queue cluster node's disk failed, the node was removed from the cluster. After restarting another node, a small portion of the metadata in the cluster was lost. What could be the reason for this?

### DISCUSSION THREAD

**Discussion by cuixianyu-github** [rabbitmq/rabbitmq-server#12343] [Other] [answered]
### Community Support Policy

- [X] I have read [RabbitMQ's Community Support Policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)

### RabbitMQ version used

Older (out of support without a commercial license)

### How is RabbitMQ deployed?

Debian package

### Steps to reproduce the behavior in question

When a queue cluster node's disk failed, the node was removed from the cluster. After restarting another node, a small portion of the metadata in the cluster was lost. What could be the reason for this?
rabbitmq_server：3.11.7
erl:25.2

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ 3.11 is [completely out of support](https://www.rabbitmq.com/release-information).

Exclusive and non-replicated queues residing on the removed node will be removed together with the node. That's the definition of "non-replicated".
